packages/extension/src/worker/service-worker.ts

```diff
@@ -25,6 +25,9 @@ import {
   fetchDictionary,
 } from '../utils/fetchCookieDictionary';
 
+//clear data from previous sessions or version of the extention
+chrome.storage.local.clear();
```

Comment:
Hmm, another clearing without any event listener when we have one on `chrome.runtime.onInstalled`? This may create confusion. Which event do we want to cover? Can we put it as part of their callbacks like `chrome.windows.onCreated `?

Reply:
I wanted to clear when the extension is refreshed ( not coverred by  )